packages/shell-bson/package.json

```diff
@@ -40,13 +40,13 @@
     "@mongosh/errors": "^2.4.4"
   },
   "peerDependencies": {
-    "bson": "^6.10.4 || ^7.0.0"
+    "bson": "^7.0.0"
```

Comment:
This makes this a `feat!` for shell-bson, probably fine to just leave this as-is for now